translators/crate.py

```diff
@@ -2,28 +2,56 @@
 from datetime import datetime, timedelta
 from translators.base_translator import BaseTranslator
 from utils.common import iter_entity_attrs
+import logging
 import statistics
 
+
 # NGSI TYPES: Not properly documented so this might change. Based on experimenting with Orion.
+NGSI_ID = 'id'
+NGSI_TYPE = 'type'
+NGSI_TEXT = 'Text'
+NGSI_DATETIME = 'DateTime'
+
 # CRATE TYPES: https://crate.io/docs/reference/sql/data_types.html
 NGSI_TO_CRATE = {
     "Array": 'array(string)',  # TODO: Support numeric arrays
     "Boolean": 'boolean',
-    "DateTime": 'timestamp',
+    NGSI_DATETIME: 'timestamp',
     "Integer": 'long',
     "geo:json": 'geo_shape',
     "Number": 'float',
-    "Text": 'string',
+    NGSI_TEXT: 'string',
     "StructuredValue": 'object'
 }
```

Comment:
we should had too: 
geo:point -> geo_point

Reply:
Created #35